src/__test__/persistentSubscription/getPersistentSubscriptionInfo.test.ts

```diff
@@ -0,0 +1,160 @@
+import { createTestNode, delay, jsonTestEvents } from "@test-utils";
+
+import {
+  END,
+  EventStoreDBClient,
+  persistentSubscriptionSettingsFromDefaults,
+  ROUND_ROBIN,
+  START,
+} from "@eventstore/db-client";
+
+describe("getPersistentSubscriptionInfo", () => {
+  const node = createTestNode();
+  let client!: EventStoreDBClient;
+
+  beforeAll(async () => {
+    await node.up();
+
+    client = new EventStoreDBClient(
+      {
+        endpoint: node.uri,
+      },
+      { rootCertificate: node.rootCertificate },
+      { username: "admin", password: "changeit" }
+    );
+  });
+
+  afterAll(async () => {
+    await node.down();
+  });
+
```

Comment:
**SUGGESTION:** I think that it may be worth checking the pessimistic case, so what will happen if we try to get status for a not-existing persistent subscription.

Reply:
Similar suggestion to tests below.